x11rb/src/extension_manager.rs

```diff
@@ -77,11 +86,22 @@ impl ExtensionManager {
         conn: &C,
         extension_name: &'static str,
     ) -> Result<Option<ExtensionInformation>, ConnectionError> {
+        let span = crate::debug_span!("Querying extension information");
```

Comment:
The general convention, IIRC, is to name the span after the function name

Reply:
Hrm... okay. Renamed the spans to the function names.